content/log/processing.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Processing
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+
+<div class="alert alert-info">
+Datadog's log management solution is is currently in private beta. If you would like to apply to it, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+
+## Overview
+
+To access the processing panel please use the upper left menu:
+{{< img src="log/processing/processing_panel.png" alt="Pipelines pannel" >}}
+
+Your processing view should look like this:
+{{< img src="log/processing/processing_view.png" alt="Pipelines view" >}}
+
+## Processing Pipelines 
+### Pipelines Goal 
+
+**A processing pipeline applies over a filtered subset of incoming logs a list of sequential processors.**
+
+With pipelines you have the opportunity to parse & enrich your logs through a sequential chains of [processors](#processors) in order to extract meaningful information or to extract attributes from semi-structured text to re-use them as [facet](/log/explore/#facets).
+
+Each log that comes through the pipelines is tested against every pipeline filter. If it matches one then all the [processors](#processors) are applied sequentially before moving to the next pipeline.
+
+So for instance a processing pipeline can transform this log:
+{{< highlight json >}}
+{
+"message":"john clicked on a link : www.datadoghq.com for app dd_app",
+"hostname":"host_1",
+"level":"info"
+}
+{{</ highlight >}}
+
+into this log: 
+
+{{< highlight json >}}
+{
+"user":"john",
+"url":"www.datadoghq.com",
+"appname":"dd_app",
+"message":"clicked on a link",
+"hostname":"host_1",
+"severity":"info"
+}
+{{< /highlight >}}
+
+With one single pipeline as follow:
+
+{{< img src="log/processing/pipeline_processors_sketch.png" alt="Pipelines processor sketch" style="width:90%;">}}
+
+Pipelines are also able to take all your logs whatever they source and transform them to handle a unifed and structured format on Datadog.
```

Comment:
> Pipelines are also able to take all your logs whatever they source and transform them to handle a unifed and structured format on Datadog.

"Pipelines can take logs from a wide variety of formats and translate them all into a common format in Datadog."

Reply:
👍